index.html

```diff
@@ -1177,119 +1177,269 @@ <h2 id="relationship-to-permissions-policy">
-          </li>
-          <li>Let |tasks| be an empty <a>list</a>.
-          </li>
-          <li>For each <a>environment settings object</a> |target| in |targets|:
-            <ol>
-              <li>
-                <a>Queue a task</a> |task| on the [=permissions task source=] of |target|'s
-                [=relevant settings object=]'s [=environment settings object/global object=]'s
-                [=Window/browsing context=] to perform the following step:
-                <ol>
-                  <li>Interpret |parameters|.{{PermissionSetParameters/state}} as if it were the
-                  result of an invocation of <a>permission state</a> for |typedDescriptor| with the
-                  argument |target| made at this moment.
-                  </li>
-                </ol>
-              </li>
-              <li>[=list/Append=] |task| to |tasks|.
-              </li>
-            </ol>
-          </li>
-          <li>Wait for all <a>tasks</a> in |tasks| to have executed.
-          </li>
-          <li>Return <a>success</a> with data `null`.
-          </li>
-        </ol>
-        <aside class="example" title="Setting a permission via WebDriver">
+        <section>
+          <h4 id="extension-module-permissions">
+            The permissions Module
+          </h4>
           <p>
-            To [=extension commands/set permission=] for `{name: "midi", sysex: true}` of the
-            [=current settings object=] of the <a>session</a> with ID 23 to "`granted`", the local
-            end would POST to `/session/23/permissions` with the body:
+            The <dfn>permissions</dfn> module contains commands for
+            managing the remote end browser permissions.
           </p>
-          <pre class="lang-json">
-          {
-            "descriptor": {
-              "name": "midi",
-              "sysex": true
-            },
-            "state": "granted"
-          }
-          </pre>
-        </aside>
+          <section>
+            <h5 id="extension-module-permissions-definition">
+              Definition
+            </h5>
+            <p>
+              [=remote end definition=]
+            </p>
+            <pre class="cddl remote-cddl">
+              PermissionsCommand = (
+                permissions.setPermission
+              )
+            </pre>
+          </section>
+          <section>
+            <h5 id="extension-module-permissions-types">
+              Types
+            </h5>
+            <section>
+              <h6 id="extension-module-type-permissions-PermissionDescriptor">
+                The permissions.PermissionDescriptor Type
+              </h6>
+              <pre class="cddl local-cddl">
+                permissions.PermissionDescriptor = {
+                  name: text,
+                }
+              </pre>
+              <p>
+                The <code>permissions.PermissionDescriptor</code> type represents a {{PermissionDescriptor}}.
```

Comment:
```suggestion
                The `permissions.PermissionDescriptor` type represents a {{PermissionDescriptor}}.
```

Reply:
Same with the others

Reply:
In theory, `<code>` and backticks are synonyms. I chose `<code>` to mirror the pattern of the BiDi spec.

In this case, I'll replace all `<code>` occurrences with backticks. Should I also replace the occurrences that this PR did not touch? (fine by me)

Reply:
Done